component/azstorage/file_share_test.go

```diff
@@ -128,6 +133,734 @@ func (s *fileTestSuite) cleanupTest() {
 	log.Destroy()
 }
 
+// func (s *fileTestSuite) TestCleanupShares() {
```

Comment:
This method is to clean up any shares in case the account has some - due to cancelling a test early or something during debug mode